nevow/test/test_appserver.py

```diff
@@ -292,3 +330,145 @@ def test_emptyPostPath(self):
             (childResource, ()), request, ('foo', 'bar'), context)
         self.assertEqual(request.prepath, [''])
         self.assertEqual(request.postpath, [])
+
+
+class OldResourceAdapterTests(TestCase):
+    def test_deferredResource(self):
```

Comment:
There are a lot of common lines between this test case and the other one in this suite.  I know `setUp` has problems, but it might make them a little easier to read if you pushed the common parts into a `setUp`.

Just a suggestion though, please feel free to ignore.